frontend/app/src/main/AndroidManifest.xml

```diff
@@ -20,7 +27,7 @@
         <activity android:name=".ViewEditListsActivity" />
         <activity android:name=".ViewOldSessionsActivity" />
         <activity android:name=".CreateNewListActivity" />
-        <activity android:name=".MainActivity"/>
+        <activity android:name=".MainActivity" />
```

Comment:
i like the consistency with the spaces paul

Reply:
That's Android Studio 😅